woocommerce/navigation
# Add Feature plugin plumbing

This PR gets the repo ready to be a feature plugin.

Note: We need linting and .editorconfig because its a bit all over the place. I'm placing "Needs review" label, but happy to wait until we get linting sorted before merging.

* Adds psr-4 
* Adds composer
* Creates FeaturePlugin and Loader classes similar to wc-admin

## Test

1. Activate this plugin.
2. Visit the plugins screen and see the message to run `composer install`.
3. `composer install`.
4. Reload the page and see "hello world" in the console.
